### PR TITLE
Homework6

### PR DESCRIPTION
**Pull request by parfenovaaa**

### CODE REVIEW

File: src/test/java/com/epam/tc/hw6/AttachmentUtils.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package com.epam.tc.hw6;
+
+import io.qameta.allure.Allure;
+import java.io.InputStream;
+//import lombok.experimental.UtilityClass;
+
+//@UtilityClass
```
Comment (akulebyakin) [inline src/test/java/com/epam/tc/hw6/AttachmentUtils.java:7]: Please remove commented imports and annotations if they are unnecessary

**Review by akulebyakin — CHANGES_REQUESTED** [review]
Please review comment

File: src/test/java/com/epam/tc/hw6/AttachmentUtils.java (hunk 1 above)
Comment (parfenovaaa) [reply src/test/java/com/epam/tc/hw6/AttachmentUtils.java]: ok

**Review by akulebyakin — APPROVED** [review]
Approved

**Pull request merged**